Library/Homebrew/locale.rb

```diff
@@ -74,6 +87,7 @@ def initialize(language, script, region)
     end
```

Comment:
Would be nice to refactor this to avoid the magic looping and set the values explicitly instead. That cannot be help avoid these all being nilable.